dev/src/container.rs

```diff
@@ -206,3 +210,32 @@ pub async fn pull_image(docker: &Docker, name: &str) -> Result<()> {
 
     Ok(())
 }
+
+fn random_lowercase_string(len: i32) -> String {
+    let mut rng = rand::thread_rng();
+    let alphabet = Uniform::from('a'..='z');
+    (0..len).map(|_| rng.sample(alphabet) as char).collect()
```

Comment:
There is already a `random_string` function in `util`, can that be used here? If it needs to be lowercase, you could change the other function to be lowercase without breaking anything.